src/main/java/data/plans/PlanList.java

```diff
@@ -416,6 +416,29 @@ public boolean checkPlanIndexIsWithinRange(int planIndex) {
         return planIndex > 0 && planIndex <= plansDisplayList.size();
     }
 
+    /**
+     * Checks whether the workout is in the plan.
+     *
+     * @param workoutToCheck        The workout to look for in the plan.
+     * @param updatedWorkoutToCheck The updated workout to look for in the plan.
+     * @param plan                  An instance of the Plan class.
+     * @return                      True if either workout or updated workout is found in the plan,
+     *                              else false if the workout is not in the plan.
+     * @throws ArrayIndexOutOfBoundsException   For operations which involves index checking.
+     */
+    public boolean checkWorkoutInPlan(String workoutToCheck, String updatedWorkoutToCheck, Plan plan)
```

Comment:
Probably you can rename the parameter for better understanding? For example, oldWorkout.

Reply:
Thank you for your suggestion.